src/App.jsx

```diff
@@ -14,57 +16,111 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {selectedGood === '' ? (
+        <h1 className="title is-flex is-align-items-center">
```

Comment:
- You don't need to compare with an empty string, just check if a value exists https://developer.mozilla.org/en-US/docs/Glossary/Falsy
- `<h1 className="title is-flex is-align-items-center">` in both cases, only content is different